vespa-http-client/src/main/java/com/yahoo/vespa/http/client/runner/CommandLineArguments.java

```
@@ -49,10 +51,18 @@ static CommandLineArguments build(String[] args) {
         if (cmdArgs.helpOption.showHelpIfRequested()) {
             return null;
         }
-
-        if (cmdArgs.hostArg == null) {
-            System.err.println("'--host' not set.");
-            return null;
+        if (cmdArgs.endpointArg != null) {
+            try {
+                URL url = new URL(cmdArgs.endpointArg);
+            } catch (MalformedURLException e) {
+                e.printStackTrace(System.err);
+                return null;
+            }
+        } else {
+            if (cmdArgs.hostArg == null) {
+                System.err.println("'--host' or '--endpoint' not set.");
+                return null;
+            }
         }
```

I think we should return an error message if both endpoint and host are set.